monitor/rpcmonitor/rpc.go

```diff
@@ -220,8 +226,8 @@ func DefaultRPCMetadataExtractor(event *EventInfo) (*policy.PURuntime, error) {
 func validateEvent(event *EventInfo) error {
 
 	if event.EventType == monitor.EventCreate || event.EventType == monitor.EventStart {
-		if len(event.Name) > 64 {
-			return fmt.Errorf("Invalid Event Name - Must not be nil or greater than 32 characters")
+		if len(event.Name) > maxEventNameLength {
+			return fmt.Errorf("Invalid Event Name - Must not be nil or greater than 64 characters")
```

Comment:
now that you have a contants, return fmt.Errorf("Invalid Event Name - Must not be nil or greater than %d characters", maxEventNameLength)